dist/encoder.js

```diff
@@ -10,6 +10,10 @@ var _web3Utils = require('web3-utils');
 
 var _web3Utils2 = _interopRequireDefault(_web3Utils);
 
+var _bn = require('bn.js');
+
+var _bn2 = _interopRequireDefault(_bn);
```

Comment:
The same question here, what's difference between `_bn` and `_bn2`

Reply:
Auto-generated by babel when building.